setup.py

```diff
@@ -28,7 +28,11 @@
     entry_points={'pytest11': ['pytest_motor = pytest_motor.plugin']},
     classifiers=[
         'Development Status :: 3 - Alpha',
-        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```

Comment:
If we are not testing against these in CI, then I am not sure that we should add them?

Reply:
Eeh... It's already saying python 3.6+ as a requirement. Guess we just need to add tests against them in CI, which I'll be doing soon with `tox` integration, discussed in #58.

Reply:
Okay, I would rather hold off on the addition of these classifiers until it is actually being tested.